app/extensions.js

```diff
@@ -408,7 +408,7 @@ module.exports.init = () => {
     extensionInfo.setInstallInfo(installInfo.id, installInfo)
     installInfo.filePath = installInfo.base_path
 
-    installInfo.base_path = chromeUrl(installInfo.base_path)
+    installInfo.base_path = fileUrl(installInfo.base_path)
```

Comment:
@kevinlawler i un-did this change from a51b7ea62c because chrome://brave URLs are not loadable in about: pages and also the code that generated context menu icons assumed a file:// URL. if i'm not mistaken, `chrome://brave` is only needed for the browser action.

Reply:
@kevinlawler 's PR before fixed the icons being broken in the top right (part of the UI Chrome).  I wonder if we need to store both the Chrome URL + file URL?

Reply:
I just verified that with this change, the icons still show up as expected 😄